website/docs/docs/build/projects.md

```diff
@@ -3,88 +3,69 @@ title: "About dbt projects"
+| [analysis-paths](/reference/project-configs/analysis-paths) | Directories to where your analyses live |
+| [macro-paths](/reference/project-configs/macro-paths) | Directories to where your macros live |
+| [snapshot-paths](/reference/project-configs/snapshot-paths) | Directories to where your snapshots live |
+| [docs-paths](/reference/project-configs/docs-paths) | Directories to where your docs blocks live |
+| [vars](/docs/build/project-variables) | Project variables you want to use for data compilation |
 
-<Tabs
-  defaultValue="cloud"
-  values={[
-    { label: 'dbt Cloud', value: 'cloud', },
-    { label: 'dbt CLI', value: 'cli', },
-  ]
-}>
-<TabItem value="cloud">
+</div>
 
-To create a new dbt project when developing in dbt Cloud:
+For complete details on project configurations, see [dbt_project.yml](/reference/dbt_project.yml).
 
-1. Create a dbt Cloud account [here](https://cloud.getdbt.com/signup/). If your organization already has a dbt Cloud account, ask an admin to add you as a Developer.
-2. If you created a new account, a new project should automatically be created. If you were added to an existing account:
-    * Click the gear icon to the top right, then **Projects** under Account Settings.
-    * Name your project, and click **Continue**. There's no need to fill in the other details.
-3. Complete the project setup flow:
-    * Connect to your <Term id="data-warehouse" />
-    * Add a repository — either choose a managed repository, or connect to an existing, but bare, repository.
+## New projects
 
-   <Lightbox src="/img/dbt-cloud-project-setup-flow-next.png" title="dbt Cloud Project Setup flow" />
+You can create new projects and [share them](/docs/collaborate/git-version-control) with other people by making them available on a hosted git repository like GitHub, GitLab, and BitBucket.
 
-4. Click **Develop** in the top left.
-5. If this is the first time you are setting up this project, then select **Initialize a project** to create your project. You will see a directory structure with `.sql` and `.yml` files generated by running the `init` command.
+After you set up a connection with your data platform, you can [initialize your new project in dbt Cloud](/docs/develop/getting-started/getting-set-up/setting-up-bigquery#initialize-your-dbt-project) and start developing. Or, run [dbt init from the command line](/reference/commands/init) to set up your new project.
 
-</TabItem>
-<TabItem value="cli">
+During project initialization, dbt creates sample model files in your project directory to help you start developing quickly.
 
-To create a new dbt project, run:
+## Sample projects
 
-```bash
-$ dbt init [project_name]
-```
+If you want to explore dbt projects more in-depth, you can clone dbt Lab’s [Jaffle shop](https://github.com/dbt-labs/jaffle_shop_duckdb) on GitHub. It's a runnable project that contains sample configurations and helpful notes.
```

Comment:
There's multiple different repos with "jaffle_shop" in the name. I've updated it below to point at the canonical one.
```suggestion
If you want to explore dbt projects more in-depth, you can clone dbt Lab’s [Jaffle shop](https://github.com/dbt-labs/jaffle_shop) on GitHub. It's a runnable project that contains sample configurations and helpful notes.
```

Reply:
fantastic, ty! i was totally wondering about that